source/includes/options-mongosqld.yaml

```diff
@@ -32,8 +32,34 @@ name: verbose
 directive: option
 aliases: -v
 description: |
-  Specifies that {{program}} should provide more detailed log output. Include
-  multiple times for more verbosity (e.g. ``-vvvvv``).
+  Specifies that {{program}} should provide more detailed log output.
+
+  You can provide this option repeatedly (up to three times) or with an
+  integer between ``1`` and ``3`` to provide greater application detail.
+  The following table describes the information provided at each log
+  level:
+
+  .. list-table::
```

Comment:
here's what we're currently working towards: https://docs.google.com/a/10gen.com/document/d/1VEzEejIrOELcHAZa2XSmiawvwTFHM0g9X5bM43OLB60/edit?usp=sharing (we also have `--quiet`)

a more detailed description of what to expect at each verbosity level is provided there